samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -371,26 +371,15 @@ Building and running
 Disabling Direction Finding feature
 ===================================
 
-To build the sample without support for the Direction Finding feature, use the following command:
-
-.. code-block:: console
-
-   west build samples/bluetooth/direct_test_mode -b board_name -- -DEXTRA_DTC_OVERLAY_FILE=no-dfe.overlay
+To build the sample without support for the Direction Finding feature, use the CMake ``EXTRA_DTC_OVERLAY_FILE`` variable to apply the :file:`no-dfe.overlay` file.
+See :ref:`cmake_options` for more information and examples of commands.
 
 Experimental HCI interface
 ==========================
 
-To build the sample with an HCI interface, use the following command:
-
-.. code-block:: console
-
-   west build samples/bluetooth/direct_test_mode -b board_name -- -DEXTRA_CONF_FILE=overlay-hci.conf
-
-On the |nRF5340DKnoref| the sample with HCI interface can also be built with the `remote_hci` image using the following command:
-
-.. code-block:: console
-
-   west build samples/bluetooth/direct_test_mode -b board_name -- -DEXTRA_CONF_FILE=overlay-hci-nrf53.conf
+To build the sample with an HCI interface, use the CMake ``EXTRA_CONF_FILE`` variable to apply the :file:`overlay-hci.conf` Kconfig overlay file.
```

Comment:
```suggestion
To build the sample with an HCI interface, use the CMake :makevar:`EXTRA_CONF_FILE` variable to apply the :file:`overlay-hci.conf` Kconfig overlay file.
```
The docs seem to have two ways of wording this instruction. I prefer this wording over the one from, for example, `samples/bluetooth/direction_finding_central/README.rst`, Line 75.

Reply:
A bit nagging about terminology around *.conf files in Teams: https://teams.microsoft.com/l/message/19:e7aa575b8c2a4438aecc6daa5b380ef8@thread.skype/1710333636958?tenantId=28e5afa2-bf6f-419a-8cf6-b31c6e9e5e8d&groupId=bbf3a6ae-b458-4b7b-9c01-2d2881d8b2fb&parentMessageId=1710333636958&teamName=NCS&channelName=General&createdTime=1710333636958